### DISCUSSION TITLE
Homarr decided to die and wont update current container or re-install

### DISCUSSION THREAD

**Discussion by Arcau** [tteck/Proxmox#1910] [Help Needed] [answered]
My LXC for Homarr decided to give up on life and when I attempted to update it get the error . 

[ERROR] in line 63: exit code 1: while executing command yarn build &> /dev/null****

Also unable to install on a fresh container. I get this returned

Loading...
bash: line 27: variables: command not found
bash: line 28: color: command not found
bash: line 29: catch_errors: command not found
bash: line 71: start: command not found
bash: line 72: build_container: command not found
bash: line 73: description: command not found
bash: line 75: msg_ok: command not found
Homarr should be reachable by going to the following URL.
         http://:3000 

using the command on debian based CT 
bash -c "$(wget -qLO - https://github.com/tteck/Proxmox/raw/main/ct/homarr.sh)"

Any idea on this at all - not super familiar with proxmox as a whole and running its containers would like to keep homarr on this box if possible - but can move it to a different OS / Box if needed :(

**Comment by tteck** [comment, marked as answer]
https://github.com/tteck/Proxmox/issues/1836
